apisix/init.lua

```diff
@@ -69,7 +72,7 @@ function _M.http_init(args)
                              "maxrecord=8000", "sizemcode=64",
                              "maxmcode=4000", "maxirconst=1000")
 
-    parse_args(args)
```

Comment:
We must keep the compatibility here, otherwise people use this way to initialize DNS resolvers will be surprised after they upgrade APISIX.

Reply:
fixed.